src/mono/cmake/config.h.in

```diff
@@ -1,6 +1,69 @@
+#ifndef __MONO_CONFIG_H__
+#define __MONO_CONFIG_H__
+
 #ifdef _MSC_VER
-#include <msvc/msvc-win32-support.h>
-#include <msvc/msvc-disabled-warnings.h>
+
+// FIXME This is all questionable but the logs are flooded and nothing else is fixing them.
```

Comment:
Is there an issue tracking this?

Reply:
Hm no, this came from msvc-disabled-warnings.h which has existed for a long time in mono/mono :)

Reply:
warnings are on other platforms as well (not msvc specific), just building with lower warning level or explicitly disabled them as part of build on other platforms.